apps/src/lib/kits/maker/portScanning.js

```diff
@@ -79,6 +80,9 @@ function listSerialDevices() {
     SerialPortType = SerialPort;
     return SerialPortType.list();
   } else {
```

Comment:
Nit: else if?

Reply:
Separately, what does it mean for this function to return a resolved promise (not sure I totally understand method signature, seems like it should return a promise that resolves to an array?)

Reply:
Can I ask the motivation behind the 'else if'? One thing I like about the way it is, is that it captures that at its core, this is a "chromebook or not" situation. Once we roll this out feature out, we will actually remove everything below the nested 'if'. Curious how that ways against the simplicity of `if/elseif/else`.

Reply:
Wait . . . I just realized we skip this whole function in the WebSerial path. Deleting in upcoming PR.